kong/router/atc_compat.lua

```diff
@@ -289,6 +304,8 @@ function _M.new(routes)
     routes = {},
     services = {},
     fields = {},
+    cache = lrucache.new(MATCH_LRUCACHE_SIZE),
```

Comment:
It will be called in `runloop/handler.lua`, and will pass two cache params, could it be compatible with this call `Router.new(routes, router_cache, router_cache_neg)`?

Reply:
I also create a PR #9148, please refer to it.